src/dev/impl/DevToys/Helpers/HashingHelper.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System;
+using System.IO;
+using System.Security.Cryptography;
+using System.Threading;
+using System.Threading.Tasks;
+using DevToys.Models;
+using DevToys.Shared.Core;
+
+namespace DevToys.Helpers
+{
+    internal static class HashingHelper
+    {
+        internal static async Task<byte[]> ComputeHashAsync(
+        HashAlgorithm hashAlgorithm,
```

Comment:
Please indent the parameters by adding 4 spaces.

Reply:
Done